frontend/packages/db-structure/src/parser/tbls/index.test.ts

```diff
@@ -14,7 +14,7 @@ describe(processor, () => {
               name: 'id',
               type: 'int',
               notNull: true,
-              primary: false, // TODO: should be true
+              primary: false,
```

Comment:
I was aiming for ``primary: true``, but since I have to add ``constraints`` to every test, I've decided to add a separate test just for primary keys.